src/main/resources/META-INF/reaktivity/http.idl

```diff
@@ -27,6 +27,7 @@ scope http
         {
             HttpHeader[] headers;
             HttpHeader[] overrides;
+            HttpHeader[] excludes;
```

Comment:
`excludes` should be a `string[]` instead of `HttpHeader[]` because it doesn't have a value.